internal/cluster/validator.go

```diff
@@ -224,6 +224,39 @@ func (v *clusterValidator) isApiVipValid(c *clusterPreprocessContext) Validation
 	return boolValue(err == nil)
 }
 
+func (v *clusterValidator) isNetworkTypeValid(c *clusterPreprocessContext) ValidationStatus {
+	validNetworkTypes := []string{models.ClusterNetworkTypeOVNKubernetes, models.ClusterNetworkTypeOpenShiftSDN}
+	if !funk.ContainsString(validNetworkTypes, swag.StringValue(c.cluster.NetworkType)) && c.cluster.NetworkType != nil {
+		return ValidationFailure
+	}
+	if network.IsIPv6CIDR(c.cluster.ClusterNetworkCidr) && swag.StringValue(c.cluster.NetworkType) != models.ClusterNetworkTypeOVNKubernetes {
```

Comment:
Other places we check also the service CIDR and cluster CIDR.  Also keep in mind none platform ->no machine CIDR

Reply:
@oshercc please fix